js/maps_lib.js

```diff
@@ -165,6 +165,7 @@
                 'address': address
             }, function (results, status) {
                 if (status === google.maps.GeocoderStatus.OK) {
+                    console.log('getgeoCondition>>>');
```

Comment:
Why are you showing this in the console? Is that related to some work in progress?